### ISSUE TITLE
'apply_query' is not defined

### ISSUE THREAD

**Issue by koriaf** [django-helpdesk/django-helpdesk#849]
This line 
https://github.com/django-helpdesk/django-helpdesk/blob/master/helpdesk/views/staff.py#L1220

raises the next error:
```
NameError /reports/{report}/
'apply_query' is not defined
```

it's related to [this](https://github.com/django-helpdesk/django-helpdesk/commit/4ca6adbe3941fefdf404baa0532e8570878f849e) commit but it seems like during some merges the old code appeared or something; I've spent some time trying to work it out but failed. 

**To Reproduce**
0. Login
1. Save some query on the tickets page
2. Navigate to the /reports/daysuntilticketclosedbymonth/
3. Load that query and apply it
4. See the exception

**Expected behavior**
The query is applied and not 500 resp is returned

The problem is applicable to all browser and OS. And the master branch at least.

**Comment by timthelion** [comment]
Just opened a PR that probably fixes this. Didn't test though.

**Comment by gwasser** [comment]
I wouldn't be surprised if I messed up a merge from the 0.2.x branch. I wanted to add some bug fixes from that branch but sometimes the diff was hard to tell where it applied because we've moved around a lot of stuff on master. I probably won't try a merge again lol. We've gotta get the 0.3 release up to date and released!

**Comment by koriaf** [comment]
Now it gives me another error:
```
File "/app/helpdesk/views/staff.py", line 30, in <module>
celerybeat_1    |     from helpdesk.query import (
celerybeat_1    | ImportError: cannot import name 'apply_query' from 'helpdesk.query' (/app/helpdesk/query.py)
```

The problem here is that apply_query was removed at some point, probably replaced by some Query object call, and lost during translation somewhere

**Comment by typonaut** [comment]
I see the same issue when attempting to run `make rundemo`

`from helpdesk.query import (
ImportError: cannot import name 'apply_query' from 'helpdesk.query' (/Users/clive/Documents/django/pipenv-01/django-helpdesk-master/helpdesk/query.py)`

**Comment by gwasser** [comment]
Yes, `master` is 0.3.0 dev. It should be Django 3.x compatible but I haven't had time to sort out the errors (deprecated features) yet so left the requirements Django 2. If you'd like to help with that, PRs welcome!

**Comment by alligatorbait** [comment]
`master` branch has this issues solved!

**Comment by gwasser** [comment]
Closing this issue since it should be fixed in `master` for the 0.3.0 release.
